examples/catalog/bin/sample_page.dart

```diff
@@ -54,14 +61,16 @@ String expandTemplate(String template, Map<String, String> values) {
   // Matches @(foo), match[1] == 'foo'
   final RegExp tokenRE = RegExp(r'@\(([\w ]+)\)', multiLine: true);
   return template.replaceAllMapped(tokenRE, (Match match) {
-    if (match.groupCount != 1)
+    if (match.groupCount != 1) {
       throw SampleError('bad template keyword $match[0]');
+    }
```

Comment:
here and many other places: why the extra braces? it's not clear that they make the code any easier to read. If anything, they just make the code longer, which generally means less fits on the screen, which makes it harder to take in everything at once.

Reply:
dartfmt would also allow these as single lines without parens. let me know if you would prefer that for `throw` and `return`

Reply:
switching the style for these.